runtime/oti/j9javaaccessflags.h

```diff
@@ -36,6 +36,8 @@
 #define J9AccBridge 0x40
 
 #define J9AccClassHasIdentity 0x20
+#define J9AccImplicitCreateHasDefaultValue 0x10
+#define J9AccImplicitCreateNonAtomic 0x20
```

Comment:
Does `J9AccImplicitCreateNonAtomic` not clash with `J9AccClassHasIdentity`?

Reply:
Good catch. I will take over fixing this.

Reply:
They are different. One is a sun modifier set in `romClass->modifiers`. The other is a J9 romclass modifier set in `romClass->extraModifiers`.

Reply:
Oh yes good point. This file is really confusing though, do you foresee this  causing any issues if I rearrange these values in a more readable way?

Reply:
The definitions ought to be organized into distinct groups so it's clearer that members of one group don't clash with members of other groups. There should be comments identifying where each group of flags is intended to be used.

Reply:
https://github.com/eclipse-openj9/openj9/issues/17785